lib/src/arithmetic/ones_complement_adder.dart

```diff
@@ -18,7 +18,7 @@ class OnesComplementAdder extends Adder {
   Logic get sign => output('sign');
 
   /// The end-around carry which should be added to the resulting [sum].
-  /// If the input [endAroundCarry] is not null, this value is stored there.
+  /// If the input `generateEndAroundCarry` is true, this value is stored there.
```

Comment:
you could make generateEndAroundCarry a `final bool generateEndAroundCarry` in the module so that you could reference it